stdlib/LinearAlgebra/src/blas.jl

```diff
@@ -1417,6 +1418,26 @@ for (fname, elty) in ((:dger_,:Float64),
     end
 end
 
+### geru
+
+"""
+    geru!(alpha, x, y, A)
+
+Performs an unconjugated rank-1 update of the matrix `A` with vectors `x` and `y` as `alpha*x*y' + A`.
```

Comment:
```suggestion
Performs an unconjugated rank-1 update of the matrix `A` with vectors `x` and `y` as `alpha*x*transpose(y) + A`.
```
I know the description already says "unconjugated" but this is the actual operation, yes?

Reply:
The altnerative would be to use `.'`.

Reply:
```julia-repl
julia> rand(3).'
ERROR: syntax: the ".'" operator is discontinued
Stacktrace:
 [1] top-level scope
   @ none:1
```